modules/oadp-plugins.adoc

```diff
@@ -29,10 +29,13 @@ OADP also provides plugins for {product-title} resource backups, OpenShift Virtu
 |`kubevirt` |Backs up and restores OpenShift Virtualization resources. ^[2]^ |Object store
 
 |`csi` |Backs up and restores volumes with CSI snapshots. ^[3]^ |Cloud storage that supports CSI snapshots
+
+|`vsm` |VolumeSnapshotMover relocates snapshots from the cluster into an object store to be used during a restore process to recover stateful applications, in situations such as cluster deletion. ^[4]^ |Object store
```

Comment:
I notice this is the only entry that explains what the plugin value stands for. Also it's a complete sentence, which breaks with the other "Function" descriptions in the table. Not sure if this is too awkward, but for style consistency maybe something like:

> Relocates snapshots from the cluster into an object store to be used during a restore process to recover stateful applications, for example during cluster deletion. Stands for VolumeSnapshotMover.

Reply:
@adellape Honestly, I need to change the other sections to provide more information. But this is part of a larger rewrite. Are you ok to accept this until I make a change? I am currently working on [improving the structure](https://github.com/openshift/openshift-docs/pull/62396) of the docs, and then will move to a rewrite
Thanks